test/Microsoft.AspNet.Razor.Runtime.Test/TagHelpers/DefaultTagHelperContentTest.cs

```diff
@@ -360,6 +457,37 @@ public void Fluent_SetContent_Append_WritesExpectedContent()
         }
 
         [Fact]
+        public void Fluent_SetContent_AppendFormat_WritesExpectedContent()
+        {
+            // Arrange
+            var tagHelperContent = new DefaultTagHelperContent();
+            var expected = "Hello World test";
+
+            // Act
+            tagHelperContent.SetContent("Hello ").AppendFormat("{0} test", "World");
+
+            // Assert
+            Assert.Equal(expected, tagHelperContent.GetContent());
+        }
+
+        [Fact]
+        public void Fluent_SetContent_AppendFormat_Append_WritesExpectedContent()
+        {
+            // Arrange
+            var tagHelperContent = new DefaultTagHelperContent();
+            var expected = "Hello World foo bar";
```

Comment:
nits:
- `var expected = new [] { "Hello ", "World foo ", "bar" };`
- should avoid `foo`, `bar`, and similar names